### PR TITLE
Dk accessibility changes

### PR DESCRIPTION
**Pull request by DavidKozdra**
Added the changes mentioned in issue #60 
included a context to avoid prop drilling the proper tab index when the modal is open 
this helps make sure when the modal is open users with mobility issues don't have to tab through the whole page
added tab index to the tiles and the formatted date and modal gradient selects

key down values are also added to make these items selectable without a mouse

A few accessibility things that can be better, the default outline on focus gets lost in some of the images bad for low vision 

include in settings a color for the outline, custom keys for submission values  fix the stop all key down seems to require a mouse value for some reason,
these would all be tracked by the context

### CODE REVIEW

**Review by riccardobertolini — APPROVED** [review]
Great job and great addition. Thanks for working on this!

File: yarn.lock
Comment (chandu) [inline yarn.lock:1]: @DavidKozdra Looks like this commit has added yarn.loc and there is an existing package-lock.json. my 2 cents, I think sticking to either yarn.lock or package-lock.json would be better for consistent dev exp.

Comment (riccardobertolini) [reply yarn.lock:1]: I haven't notice that, good catch Chandu.
Preparing now a PR to remove it - thanks!

Comment (DavidKozdra) [reply yarn.lock:1]: sorry I didn't see all this thought we defulted with yarn I see now that the read me is using npm my bad

**Pull request merged**